xcm/xcm-executor/src/lib.rs

```diff
@@ -272,13 +272,13 @@ impl From<ExecutorError> for frame_benchmarking::BenchmarkError {
 }
 
 impl<Config: config::Config> XcmExecutor<Config> {
-	pub fn new(origin: impl Into<MultiLocation>, message_hash: XcmHash) -> Self {
-		let origin = origin.into();
+	pub fn new(computed_origin: impl Into<MultiLocation>, message_hash: XcmHash) -> Self {
+		let computed_origin = computed_origin.into();
```

Comment:
This is not the computed origin -- whatever that gets sent in is definitely the physical origin.

Reply:
makes sense. I've updated it